src/Strategy/Go/Types.hs

```diff
@@ -59,8 +60,8 @@ golangPackageToDependency pkg = foldr applyLabel start
 
 -- replace "v0.0.0-20191212000000-abcdef+incompatible" with "abcdef"
 fixVersion :: Text -> Text
-fixVersion = last . T.splitOn "-" . T.replace "+incompatible" ""
+fixVersion = Unsafe.last . T.splitOn "-" . T.replace "+incompatible" ""
```

Comment:
Using `last` here is safe because `splitOn` _always_ returns a non-empty list.

(I wish it'd return a `Data.List.NonEmpty` instead)